### PR TITLE
Changes from pr2_benchmarking work

### PR DESCRIPTION
**Pull request by davetcoleman**
- Added TRRT to PR2's planning option
- Added a whole body planning group config
- Added launch file for using the Benchmark GUI
- Tweaked/cleaned up launch files in pr2_moveit_config

### CODE REVIEW

File: pr2_moveit_config/config/ompl_planning.yaml [hunk 1]
```diff
@@ -88,6 +107,13 @@ planner_configs:
     type: geometric::RRT
     range: 100000000
 
+  TRRTkConfigDefault:
+    type: geometric::RRT
+    range: 100000000
```
Comment (isucan) [inline pr2_moveit_config/config/ompl_planning.yaml]: are we sure this is correct?

File: pr2_moveit_config/config/ompl_planning.yaml [hunk 2]
```diff
@@ -49,6 +52,7 @@ left_arm_and_torso:
     - SBLkConfigDefault 
     - LBKPIECEkConfigDefault
     - RRTkConfigDefault
+    - TRRTkConfigDefault
```
Comment (isucan) [inline pr2_moveit_config/config/ompl_planning.yaml:55]: we should change the setup_assistant template to add this planner too.

File: pr2_moveit_config/config/ompl_planning.yaml (hunk 1 above)
Comment (davetcoleman) [inline pr2_moveit_config/config/ompl_planning.yaml]: Absolutely not :) I'll remove the range

File: pr2_moveit_config/config/ompl_planning.yaml (hunk 2 above)
Comment (davetcoleman) [inline pr2_moveit_config/config/ompl_planning.yaml:55]: created issue.

**Pull request merged**